packages/@o3r/training-sdk/jest.config.js

```diff
@@ -0,0 +1,26 @@
+/** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
+module.exports = {
+  displayName: require('./package.json').name,
+  preset: 'ts-jest',
+  rootDir: '.',
+  roots: ['<rootDir>/src/'],
+  reporters: [
+    'default',
+    ['jest-junit', { outputDirectory: '<rootDir>/dist-test', suiteName: '@o3r/training-sdk unit tests' }]
+  ],
+  moduleNameMapper: {
```

Comment:
Is the monorepo helper not used on purpose?

Reply:
probably not